components/fxa-client/ios/FxAClient/FxAccountManager.swift

```diff
@@ -647,14 +647,8 @@ public struct FxaAuthData {
     }
 }
 
-public struct DeviceConfig {
-    let name: String
-    let type: DeviceType
-    let capabilities: [DeviceCapability]
-
+extension DeviceConfig {
     public init(name: String, type: DeviceType, capabilities: [DeviceCapability]) {
-        self.name = name
-        self.type = type
-        self.capabilities = capabilities
+        self.init(name: name, deviceType: type, capabilities: capabilities)
```

Comment:
I needed to change the field name since `type` isn't allowed in Rust.  I think this means that this won't be a breaking change for firefox-ios, but please double-check that.

Reply:
We run our new branch builds to verify!